src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ListViewTests/UnoSamples_Tests.ListView.cs

```diff
@@ -163,5 +158,166 @@ public void ListView_ObservableCollection_Unused_Space()
+
+			var checkBoxHeader = _app.Marked("CheckBoxHeader");
+			_app.WaitForElement(checkBoxHeader);
+
+			// Save initial state(not expanded)
+			var screenshot1 = _app.Screenshot("ListView Expanded Control - Initial State");
+
+			// Expand and compare
+			checkBoxHeader.Tap();
+			var screenshot2 = _app.Screenshot("ListView Expanded Control - expanded State");
+			if (!UITestHelper.CompareScreenShots(screenshot1, screenshot2))
+			{
+				Assert.Fail("Expand is not working");
+			}
+
+			// Collapse and compare 
+			checkBoxHeader.Tap();
+			var screenshot3 = _app.Screenshot("ListView Expanded Control - collapsed State");
+			if (UITestHelper.CompareScreenShots(screenshot1, screenshot3))
+			{
+				Assert.Fail("Collapse is not working");
+			}
+		}
+		[Test]
+		[AutoRetry]
+		public void ListView_ExpandableItemLarge_ExpandHeaderWithMultipleItems_Validation()
+		{
+			Run("SamplesApp.Windows_UI_Xaml_Controls.ListView.ListView_Expandable_Item_Large");
+
+			var checkBoxHeader = _app.Marked("CheckBoxHeader");
+			_app.WaitForElement(checkBoxHeader);
+
+			// Save initial state(not expanded)
+			var screenshot1 = _app.Screenshot("ListView Expanded Control - Initial State");
+
+			// Expand and compare
+			checkBoxHeader.Tap();
+			ClickCheckBoxAt(0);
+			ClickCheckBoxAt(1);
+			ClickCheckBoxAt(2);
+			var screenshot2 = _app.Screenshot("ListView Expanded Control - expanded State");
+			if (!UITestHelper.CompareScreenShots(screenshot1, screenshot2))
+			{
+				Assert.Fail("Expand is not working");
+			}
+
+			// Collapse and compare
+			checkBoxHeader.Tap();
+			ClickCheckBoxAt(0);
+			ClickCheckBoxAt(1);
+			ClickCheckBoxAt(2);
+			var screenshot3 = _app.Screenshot("ListView Expanded Control - collapsed State");
+			if (UITestHelper.CompareScreenShots(screenshot1, screenshot3))
+			{
+				Assert.Fail("Collapse is not working");
+			}
+		}
+
+		[Test]
+		[AutoRetry]
+		public void ListView_ExpandableItemLarge_ExpandHeaderWithSingleItem_Validation()
+		{
+			Run("SamplesApp.Windows_UI_Xaml_Controls.ListView.ListView_Expandable_Item_Large");
+
+			var checkBoxHeader = _app.Marked("CheckBoxHeader");
+			_app.WaitForElement(checkBoxHeader);
+
+			// Save initial state(not expanded)
+			var screenshot1 = _app.Screenshot("ListView Expanded Control - Initial State");
+
+			// Expand multiple items, header and compare
+			checkBoxHeader.Tap();
+			ClickCheckBoxAt(0);
+			var screenshot2 = _app.Screenshot("ListView Expanded Control - expanded State");
+			if (!UITestHelper.CompareScreenShots(screenshot1, screenshot2))
+			{
+				Assert.Fail("Expand is not working");
+			}
+
+			// Collapse all and compare 
+			checkBoxHeader.Tap();
+			ClickCheckBoxAt(0);
+			var screenshot3 = _app.Screenshot("ListView Expanded Control - collapsed State");
+			if (UITestHelper.CompareScreenShots(screenshot1, screenshot3))
+			{
+				Assert.Fail("Collapse is not working");
+			}
+		}
+
+		private void ClickCheckBoxAt(int i)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make 'ClickCheckBoxAt' a static method.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826516)

Reply:
This can't be a static method, robo-Carl.

Reply:
Can we put "resolved" for this conversation @davidjohnoliver ?

Reply:
I think you can safely resolve/ignore all comments made by robo-Carl during its brief reign of terror. So, yes.

Reply:
Thanks for the confirmation @davidjohnoliver ;) !